props/saber_fett263_buttons.h

```diff
@@ -411,12 +441,18 @@ SaberFett263Buttons() : PropBase() {}
 #endif
         if (!swinging_) {
           swing_blast_ = false;
-#ifndef FETT263_BM_DISABLE_OFF_BUTTON
+#ifdef FETT263_BM_DISABLE_OFF_BUTTON
+        if (!battle_mode_) {
           Off();
           saber_off_time_ = millis();
+        }
+#else
+         Off();
+         saber_off_time_ = millis();
+         battle_mode_ = false;
 #endif
-#ifndef FETT263_BATTLE_MODE_ALWAYS_ON
-          battle_mode_ = false;
+#ifdef FETT263_BATTLE_MODE_ALWAYS_ON
```

Comment:
Did you mean to change this back again?
Which is it supposed to be?

Reply:
this was tied to the previous change that was merged to handle the button changing to false.